web/src/app/views/post.view.html

```diff
@@ -102,36 +102,45 @@ <h4 class="title is-4 is-spaced">
 
         <h5 class="subtitle is-5">All fields are required &middot; Your message will be sent to the poster by email</h5>
 
-        <form>
+        <form ng-submit="ctrl.submit()">
+
+          <div class="notification-wrapper" ng-show="ctrl.errors.general !== undefined">
+            <div class="notification is-danger">
+              <button type="button" class="delete" ng-click="ctrl.errors.general = undefined"></button>
+              <p class="subtitle">{{ ctrl.errors.general }}</p>
+            </div>
+          </div>
 
           <div class="field">
             <div class="field-body">
               <div class="field">
                 <p class="control is-expanded has-icons-left">
-                  <input class="input is-medium" type="text" placeholder="Full name">
```

Comment:
Should we keep full name requirement for replies?

Reply:
I dont mind, what's the thinking behind doing that?